config/locales/en/devise.yml

```diff
@@ -19,6 +19,9 @@ en:
     mailer:
       confirmation_instructions:
         subject: 'Confirmation instructions'
+        welcome: 'Welcome %{email} !'
+        instruction: 'You can confirm your email through the link below:'
```

Comment:
@lowjoel I know you have concerns about `:` at the end of the sentence, but in the devise mailer, they put these whole string in a paragraph, the final format is like:
```
<p>Welcome test@example.org !</p>

<p>You can confirm your email through the link below:</p>

<p><a href="http://localhost:3000/users/confirmation?confirmation_token=pdsMptRq23ymaxWg8uWR">Confirm my email</a></p>
```